easybuild/easyconfigs/g/GAlib/GAlib-2.4.7-GCC-14.3.0.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'GAlib'
+version = '2.4.7'
+local_short_version = version.replace('.', '')
+
+homepage = 'https://web.mit.edu/galib'
+description = """GAlib is a set of C++ genetic algorithm objects.
+The library includes tools for using genetic algorithms to do optimization in any C++ program.
+"""
+
+toolchain = {'name': 'GCC', 'version': '14.3.0'}
+
+source_urls = ['http://lancet.mit.edu/ga/dist/']
+sources = [f'%(namelower)s{local_short_version}.tgz']
+checksums = ['ea76b66ce4db4db2ed86e20d6d3ff144abaf73e33620104246639d9b2a465329']
+
+skipsteps = ['configure']
+
+prebuildopts = "sed -i -e 's|-g -Wall|-Wall -fPIC -fpermissive -std=c++98 -O3|g' "
```

use `$CXXFLAGS`:
```suggestion
prebuildopts = 'sed -i -e "s|-g -Wall|-Wall $CXXFLAGS|g" '
```

and also use `toolchainopts`:
```python
toolchainopts = {'pic': True, 'extra_cxxflags': '-fpermissive', 'cstd': 'c++98'}